server/models/Group.js

```diff
@@ -248,11 +248,27 @@ export default function(Sequelize, DataTypes) {
       },
 
       addUserWithRole(user, role) {
-        return Sequelize.models.UserGroup.create({
-          role,
-          UserId: user.id,
-          GroupId: this.id
-        });
+        const lists = {};
+        lists[roles.BACKER] = 'backers';
+        lists[roles.MEMBER] = 'members';
+        lists[roles.HOST] = 'host';
+
+        const notifications = [{type:`mailinglist.${lists[role]}`}];
+
+        switch (role) {
+          case roles.HOST:
+            notifications.push({type:activities.GROUP_TRANSACTION_CREATED});
+            notifications.push({type:activities.GROUP_EXPENSE_CREATED});
```

Comment:
That's a new behavior. I agree with it but I'd do another PR for it. And we will have to communicate about it. Also, not sure the members can approve expense in the current code (and need to make sure that one cannot approve own expense)